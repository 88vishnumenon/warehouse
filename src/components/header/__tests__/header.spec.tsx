import { render,cleanup } from "@testing-library/react";
import { ThemeProvider } from '@material-ui/styles';

import customTheme from '../../../assets/theme/custom-theme';

import Header from "../header"


function renderHeader() {
    return render(
        <ThemeProvider theme={customTheme}>
        <Header></Header>
        </ThemeProvider>
    );
}

afterEach(() => {
    jest.restoreAllMocks();
    cleanup();
});

describe("<Header /> test utils", () => {
    // snapshots
    test('should match snapshot', () => {
        const { asFragment } = renderHeader();
        expect(asFragment()).toMatchSnapshot();
    });

})