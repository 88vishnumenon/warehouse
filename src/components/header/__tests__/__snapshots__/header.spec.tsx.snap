// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> test utils should match snapshot 1`] = `
<DocumentFragment>
  <section
    class="makeStyles-header-1"
  >
    <img
      class="makeStyles-logo-2"
      src="/assets/images/logo.png"
    />
  </section>
</DocumentFragment>
`;
